src/modules/internal_resistance/internal_resistance_params.c

```diff
@@ -0,0 +1,2 @@
+PARAM_DEFINE_INT32(INTER_RES_EN, 1);
```

Comment:
Can you add comments (metadata) like the other param files? That's where the group, min, max, etc come from in QGC.